src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figure;
+
+import base.Figure;
+
+public class RightTriangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration (line 4).